hw/ip/otp_ctrl/data/BUILD

```diff
@@ -24,6 +24,11 @@ int_flag(
     build_setting_default = 0,
 )
 
+int_flag(
```

Comment:
you will also need to add an attribute to the `otp_image` rule ([here](https://cs.opensource.google/opentitan/opentitan/+/master:rules/otp.bzl;drc=a0da3fcc689d6d26859858e84d33ae32f32cd756;l=202) and [here](https://cs.opensource.google/opentitan/opentitan/+/master:rules/otp.bzl;drc=a0da3fcc689d6d26859858e84d33ae32f32cd756;l=160)) that uses this build flag, and pipes it through Bazel to the invocation of the image generation Python script

Reply:
Ah that is what was missing yes. Thanks for the pointers.